src/native/_mutable.js

```diff
@@ -65,7 +65,7 @@ const MDataInfo = Struct({
   /// Name of the mutable data.
   name: t.XOR_NAME,
   /// Type tag of the mutable data.
-  type_tag: t.u64,
+  typeTag: t.u64,
```

Comment:
I'm not sure if it really matters but perhaps is worthy keeping the prop name here as `type_tag` as it's supposed to match the native struct.